examples/nativemodules/PureNativeModule.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { fireEvent, render } from 'react-native-testing-library';
+import PureNativeModule from './PureNativeModule';
+
+jest.mock('react-native', () => {
+  const RN = jest.requireActual('react-native');
+  RN.NativeModules.NativeNotification = {
+    show: jest.fn(),
+  };
```

Comment:
Some native components, like `react-native-safe-area-context`, register through `UIManager.getViewManagerConfig`. We need to add that as well. Check their tests:
https://github.com/th3rdwave/react-native-safe-area-context/blob/24018ffa38fec85fb5dac1f34a93d23319f7df7b/src/__tests__/initialWindowMetrics-test.tsx